sirepo/auth/__init__.py

```diff
@@ -135,6 +135,27 @@ def complete_registration(name=None):
     cookie.set_value(_COOKIE_STATE, _STATE_LOGGED_IN)
 
 
+def control_sim_type_role(sim_type):
+    from sirepo import oauth
+    from sirepo import auth_role_moderation
+
+    t = sirepo.template.assert_sim_type(sim_type)
+    if t not in sirepo.feature_config.auth_controlled_sim_types():
+        return
+#QUESTION(robnagler) I think this is sufficient, that is, the tests can be reversed
```

Comment:
This is the only thing I'm concerned about. LMK what you think.

Reply:
That doesn't workin in the `api_checkAuthJupyterhub` case. That API is `allow_visitor` which means that this check is always True and we return when we should really be prompting to moderate. 

Thoughts ccaa8f691 ? I don't love adding another api_perm and I struggled to find a name for it...